python/composio/cli/workspaces.py

```diff
@@ -0,0 +1,138 @@
+"""
+Composio workspace manager.
+"""
+
+import typing as t
+from datetime import datetime
+from uuid import uuid4
+
+import click
+
+from composio.cli.context import Context, pass_context
+from composio.cli.utils.decorators import handle_exceptions
+from composio.cli.utils.params import EnumParam
+from composio.client.collections import ComposioWorkspaceStatus
+from composio.client.exceptions import HTTPError
+
+
+@click.group(name="workspaces", invoke_without_command=True)
+@click.option(
+    "--status",
```

Comment:
The method `_get_workspace_id` in `workspaces.py` uses the variable `id` which might be `None` and still raises an exception with the message indicating that a workspace with that ID was not found. This could be misleading if `id` is actually `None` because it implies an ID was searched for but not found, rather than no ID being provided at all.